lua/astrocommunity/pack/typescript/biome.lua

```diff
@@ -0,0 +1,32 @@
+local has_biome = function()
+  local bufnr = vim.api.nvim_get_current_buf()
+  local rooter = require "astrocore.rooter"
+
+  local biomejson_rooter = rooter.resolve {
+    "biome.json",
+  }
+
+  return next(biomejson_rooter(bufnr))
+end
+
+return {
+  { import = "astrocommunity.pack.typescript.core" },
+  {
+    "williamboman/mason-lspconfig.nvim",
+    opts = function(_, opts)
+      opts.ensure_installed = require("astrocore").list_insert_unique(opts.ensure_installed, { "biome" })
+    end,
+  },
+  {
+    "jay-babu/mason-null-ls.nvim",
+    opts = function(_, opts)
+      if not opts.handlers then opts.handlers = {} end
+
+      if not has_biome() then return end
```

Comment:
Not really happy about this. I tried to use `get_clients` to just check if biome is attached, but in this case it didn't seem feasible because the lsp is attaching after this config is running.